tests/integration/test_wild.py

```diff
@@ -76,7 +76,7 @@ def test_amazon_doctype(tmpdir):
     # amazon gzips its homepage.  For some reason, in requests 2.7, it's not
     # getting gunzipped.
     with vcr.use_cassette(str(tmpdir.join('amz.yml'))):
-        r = requests.get('http://www.amazon.com')
+        r = requests.get('http://www.amazon.com', verify=False)
```

Comment:
Amazon.com now redirects to https, which is why this test started failing.  Since this test is doing nothing where SSL/TLS is actually needed, I think setting `verify=False` is fine.

For posterity's sake, the underlying reason why this is happening is because `runtests.sh` overrides `REQUESTS_CA_BUNDLE`, needed for some httpbin TLS tests.  This combined with Amazon redirecting is what caused tests to stop failing.